core/src/overlay_client/neighbours.rs

```diff
@@ -65,7 +65,7 @@ impl Neighbours {
         index
             .indices_with_weights
             .sort_by(|(_, lw), (_, rw)| rw.cmp(lw));
-        return Vec::from(index.indices_with_weights.as_slice());
+        Vec::from(index.indices_with_weights.as_slice())
```

Comment:
`index.indices_with_weights.to_vec()` ?

Reply:
> `index.indices_with_weights.to_vec()` ?

I'll make a follow-up pr, it has a lot of space for improvement

Reply:
https://github.com/broxus/tycho/pull/477